pkg/bindings/manifests/manifests.go

```diff
@@ -71,7 +71,7 @@ func Exists(ctx context.Context, name string, options *ExistsOptions) (bool, err
 }
 
 // Inspect returns a manifest list for a given name.
-func Inspect(ctx context.Context, name string, _ *InspectOptions) (*manifest.Schema2List, error) {
+func Inspect(ctx context.Context, name string, _ *InspectOptions) (*libimage.ManifestListData, error) {
```

Comment:
Sorry but we have to revert this, we cannot allow any breaking changes in pkg/bindings since this is supported for external uses as well

Reply:
Apologies for not catching that during the review, @sstosh.

What we probably need to do is to create a new function (and new option type). `func InspectListData` for instance.